docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaSubmissionQueueHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\WebformHandler;
+
+use Drupal\webform\Plugin\WebformHandler\EmailWebformHandler;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * Emails a webform submission.
+ *
+ * @WebformHandler(
+ *   id = "foia_submission_queue",
+ *   label = @Translation("FOIA Submission Queue"),
+ *   category = @Translation("Queueing"),
+ *   description = @Translation("Queues a webform submission to be sent later."),
+ *   cardinality = \Drupal\webform\WebformHandlerInterface::CARDINALITY_UNLIMITED,
+ *   results = \Drupal\webform\WebformHandlerInterface::RESULTS_PROCESSED,
+ * )
+ */
+class FoiaSubmissionQueueHandler extends EmailWebformHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function sendMessage(WebformSubmissionInterface $webform_submission, array $message) {
+    // @var @var QueueFactory $queue_factory
+    $queue_factory = \Drupal::service('queue');
+
+    // @var QueueInterface $queue
+    $queue = $queue_factory->get('foia_queue_submissions');
```

Comment:
Maybe call this something like `foia_submissions_queue` or just `foia_submissions`. The current name sounds kind of like an action.